include/bitcoin/explorer/commands/qrcode.hpp

```diff
@@ -125,7 +127,8 @@ class BCX_API qrcode
     virtual arguments_metadata& load_arguments()
     {
         return get_argument_metadata()
-            .add("PAYMENT_ADDRESS", 1);
+            .add("PAYMENT_ADDRESS", 1)
+            .add("filename", 1);
```

Comment:
This output was from the shell command 'file out2.png'.  The command itself outputs nothing on success so that the return code can be checked.